File: 1065.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int s, c, c1;
    s = 0;
    for(c = 1; c < 6; c++){
        cin >> c1;
        if(c1 % 2 == 0){
            s += 1;
        }
    }
    cout << s << " valores pares" << endl;
    return 0;
}

File: 1065.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        int s = 0;
        for(int c = 1; c < 6; c++) {
                int h = input.nextInt();
                if((h % 2) == 0) {
                        s += 1;
                }
        }
        System.out.println(s+" valores pares");
}
}
